_extras/demo_lessons.md

```diff
@@ -3,63 +3,102 @@ layout: page
 title: "Suggested Lessons for Teaching Demonstrations"
 ---
 
-Please see the tables below for recommendations about choosing a lesson for your teaching demonstration. If you would like to use a lesson that is not listed here, please contact the Instructor Trainer leading your teaching demonstration or (if no one is listed for your session) email [checkout@carpentries.org](mailto:checkout@carpentries.org).  Note that while a trainee can choose the lesson to teach from, the Trainer leading the demo session will choose the specific episode at the time of the demo session.
+The list below shows lessons that have been reviewed for good and bad start points to use for a teaching demonstration. When selecting a lesson for a 
+teaching demonstration, trainees should review this list to ensure it is suitable.
 
-If the chosen episode is very short and finishes in less than five minutes, the trainee can continue on to the next episode in that lesson.
+Those who wish may also consult the detailed tables following the list to identify episodes marked "avoid" because they will not be asked to teach those episodes. However, we recommend 
+reviewing the full lesson regardless, as this will help provide context for other start points and is also good preparation for your eventual teaching 
+role!
+
+Note that while a trainee can choose the lesson to teach from, the **Trainer leading the demo session will choose the specific episode at the time 
+of the demo session.** If you would like to use a lesson that is not listed here, you may contact the Instructor Trainer leading your teaching 
+demonstration or (if no one is listed for your session) email [checkout@carpentries.org](mailto:checkout@carpentries.org).  
```

Comment:
while this is open should this address be changed? @karenword

Reply:
Yes, it should be changed to instructor.training@carpentries.org